babel/src/main/java/org/apache/calcite/sql/babel/SqlColumnAttributeGenerated.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.sql.SqlColumnAttribute;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * A {@code SqlColumnAttributeGenerated} represents a column
+ * attribute specified by the GENERATED statement.
+ */
+public class SqlColumnAttributeGenerated extends SqlColumnAttribute {
+
+  public final GeneratedType generatedType;
+  public final List<SqlColumnAttributeGeneratedOption> generateOptions;
```

Comment:
nit: generatedOptions to be consistent with the class type? (here and elsewhere in the file).